tools/integration-tests/tester/docker-compose.yml

```diff
@@ -3,11 +3,14 @@ version: "3.5"
 services:
   tester:
     container_name: tester
-    image: golang:1.16.2
+    image: golang:1.16
```

Comment:
👍 

Just want to say that one day a bug may pop up from a silent update of a minor version of some shitty lib (probably not go), and it would be hard to track where it came from.